IFC4x3/PropertyConstants/w/WELDEDANDNOTINSERTABLETOCONTINOUSWELDEDRAIL_2PQcH$tHvFWx4R4rSZpvt5.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="WELDEDANDNOTINSERTABLETOCONTINOUSWELDEDRAIL" UniqueId="996a647f-dd1e-4f83-b11b-135723cf9dc5" />
```

Comment:
Difficult to read.

Reply:
Done : 426664c10f4b7a9f9b1eff4dddaa4ee62a65d0e3